index.html

```diff
@@ -888,6 +888,26 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
           in the same server as the controlled resource.
         </p>
       </section>
+      <section id="webids">
+        <h2>WebIDs</h2>
+        <blockquote class="informative">
+          <p>
+            Non-normative note:
+            The [[!SOLIDWEBAC]] <a href="https://github.com/solid/web-access-control-spec#overview">overview</a>
+            mentions parenthetically that user and group identifiers should be WebIDs.  For the purposes of performing
+            authorization, the essential requirements are:
+          </p>
+          <ul>
+            <li>
+              User and group identifiers must be IRIs.
```

Comment:
One key piece about WebIDs is that they are not just IRIs but also URLs. Is that requirement not part of this? Requiring a URL is implied below in that "Group IRIs must be dereferenceable", so would it make sense to state that more explicitly? And I assume from this that User IRIs don't necessarily need to be URLs, correct?